src/m3ninx/postings/roaring/roaring.go

```diff
@@ -37,9 +36,8 @@ var (
 	errIteratorClosed        = errors.New("iterator has been closed")
 )
 
-// postingsList wraps a Roaring Bitmap with a mutex for thread safety.
+// postingsList wraps a Roaring Bitmap with the m3ninx pl API.
```

Comment:
worth calling out that it's not goroutine-safe?

Reply:
Sure, will do in a follow up change.